inabox/deploy/config_types.go

```diff
@@ -167,7 +175,8 @@ type Config struct {
 
 	EigenDA           EigenDAContract     `yaml:"eigenda"`
 	BlobVersionParams []*BlobVersionParam `yaml:"blobVersions"`
-	MockRollup        string              `yaml:"mockRollup" json:"mockRollup"`
+	EigenDAV1CertVerifier        string              `yaml:"v1CertVerifier" json:"v1CertVerifier"`
+	EigenDAV2CertVerifier        string              `yaml:"v2CertVerifier" json:"v2CertVerifier"`
```

Comment:
again why some have both yaml and json, some only yaml?

Reply:
you're asking the wrong person haha